### PR TITLE
First draft of February newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
February newsletter :) Ready for review!

### CODE REVIEW

File: docs/blog/newsletter-february-2019.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+#########################################
+Write the Docs Newsletter – February 2019
+#########################################
+
+Hello, Write the Docs people! Beth here, wishing you all a happy 2019. Is it too late in the year for that? Oh well, never mind. I'm doing it anyway.
+
+We've got a little something for you this month. There's tons of useful stuff hidden away in the `newsletter archive </blog/archive/tag/newsletter/>`_, and it's a shame that it's not very discoverable. So, in an appropriately documentarian fashion, we've had a go at fixing that by creating an `index of all the topics </newsletter/#index-of-newsletter-topics>`_ covered by the newsletter over the last few years. We hope you like it!
+
+In other news, on the conference front: we now have `dates for the 2019 Prague conference </conf/prague/2019/news/prague-19-dates/>`_! It'll be 15th-17th September, back in the gorgeous Auto Klub. Ticket sales and call for proposals should open in March - keep your eyes peeled.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2019.rst:12]:
```suggestion
In other news, on the conference front we now have `dates for the 2019 Prague conference </conf/prague/2019/news/prague-19-dates/>`_! It'll be 15th-17th September, back in the gorgeous Auto Klub. Ticket sales and call for proposals should open in March - keep your eyes peeled.
```

File: docs/blog/newsletter-february-2019.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Onto our stories! What have the community been talking about on Slack recently?
+
+--------------------------------------
+Agile delivery and continuous releases
+--------------------------------------
+
+One topic that came up this month: how do you successfully deliver documentation when features roll out continuously... as in every two weeks?
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2019.rst]: Aren't all these topics that came up this month? :-p

File: docs/blog/newsletter-february-2019.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+* Keep an eye on future documentation needs. Join backlog refinement meetings and get the product roadmap so you know what's coming up.
+* Keep release notes brief. Capture feature details in a separate document that you can develop into full documentation when the time is right.
+* Communicate documentation status to users. If code has shipped but docs are in progress, make it clear that documentation is being updated. Be explicit when a feature is released but still in testing.
+* One suggestion was to follow the `Azure DevOps release notes process <https://channel9.msdn.com/Blogs/DevOps-Interviews/Interview-with-Aaron-Bjork-Release-Notes>`_.
+
+---------------------------------------------
+Personal development goals for documentarians
+---------------------------------------------
+
+A recent question about personal, work-related development goals produced quite a thread, and brought many common interests to the fore. Here are some of the areas the community wants to develop in 2019 - we are evidently a group driven to expand our horizons!
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2019.rst]: This implies to me that the community wants to develop them collaboratively, which isn't the case here. Right?

File: docs/blog/newsletter-february-2019.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+* **UI/UX**. Several people wanted either to move in this direction in their careers, or to polish their skills in their present positions.
+
+* **Public speaking**. This one really resonated, with some folks pursuing Toastmasters, others looking to give presentations on specific topics, and still others committing to submit proposals for conferences.
+
+* **New languages**. Not just programming languages, but human languages too! We're an international community with contributors who work for international companies, and language skills can really help bridge communication gaps.
+
+* Other ideas included working on graphics and illustration skills, contributing to tech communities (new meetups, new conferences, new outreach groups), working more deeply with the products you document - especially if they're for developers or sysadmins - and so many more!
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2019.rst]:
```suggestion
* Other ideas included working on graphics and illustration skills, contributing to tech communities (new meetups, new conferences, new outreach groups), and working more deeply with the products you document - especially if they're for developers or sysadmins!
```

File: docs/blog/newsletter-february-2019.rst [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+We did discuss a few ways that you could approach writing that documentation: 
+
+* "By default, deletes all objects. Optionally, ..."
+* ":exclamation: Warning :exclamation: Be careful here!"
+* Using all caps and bold to draw attention.
+
+But really, the design needs to be improved. There are safer approaches to bulk deletion: for example, the endpoint could require a filter parameter, and allow a wildcard value that allows bulk deletion. Users can still bulk delete, but this makes it a more deliberate action. 
+
+The takeaway is that you cannot save a bad design with documentation. Not even by documenting the bad design in bold and all caps.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2019.rst:80]: <3

File: docs/newsletter.rst [hunk 6]
```diff
@@ -53,6 +53,7 @@ and some topics are repeated if they fit into more than one category. Enjoy!
 Collaboration
 -------------
```
Comment (plaindocs) [inline docs/newsletter.rst:54]: AND on the fly categorization. :tada:

**Review by plaindocs — CHANGES_REQUESTED** [review]
Persnickety changes only.

File: docs/blog/newsletter-february-2019.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2019.rst:12]: The colon makes sense, doesn't it?

File: docs/blog/newsletter-february-2019.rst (hunk 2 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2019.rst]: Fair point!

**Review by hillaryfraley — DISMISSED** [review]
Beth, I made my suggestions directly in the Jan 31 commit.

File: docs/blog/newsletter-february-2019.rst (hunk 1 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2019.rst:12]: Yeah, i think so. I don't see that it adds anything over it not being there though? But probably pure personal preference, so disregard. :man_shrugging:

**Review by plaindocs — APPROVED** [review]

**Pull request merged**